services/app-web/src/pages/App/AppRoutes.tsx

```diff
@@ -67,10 +72,26 @@ const StateUserRoutes = ({
                     path={RoutesRecord.SUBMISSIONS_NEW}
                     element={<NewStateSubmissionForm />}
                 />
-                <Route
-                    path={RoutesRecord.SUBMISSIONS_SUMMARY}
-                    element={<SubmissionSummary />}
-                />
+                <Route element={<SubmissionSideNav />}>
+                    {showQuestionsAnswers && (
+                        <Route
+                            path={
+                                RoutesRecord.SUBMISSIONS_QUESTIONS_AND_ANSWERS
+                            }
+                            element={<QuestionsAnswers />}
+                        />
+                    )}
+                    <Route
+                        path={RoutesRecord.SUBMISSIONS_SUMMARY}
+                        element={<SubmissionSummary />}
+                    />
+                </Route>
+                {showQuestionsAnswers && (
```

Comment:
We wouldn't be able to wrap `SUBMISSIONS_UPLOAD_QUESTION` and `SUBMISSIONS_QUESTIONS_AND_ANSWERS` together because `SUBMISSIONS_QUESTIONS_AND_ANSWERS` is a nested route inside the `SubmissionSideNav` layout route.

Reply:
Oh yes! Didn't see that.